docs/components/validation/validators.md

```diff
@@ -673,6 +673,60 @@ Skriv så här i kod när användaren ska fylla i personnummer:
 
 Felmeddelandet till användarenn när valideringen inte är godkänd är:<br> Kolla att personnumret stämmer.
 
+### Personnummer - inte samma `personnummerNotSame`
+
+Validatorn kontrollerar att det ifyllda personnumret inte är samma som personnumret i annat fält.
+
+Det finns en separat validator för att kontrollera format för personnummer, personnummer-format (`personnummerFormat`). Validatorn för format ska stå före validatorn för `personnummerNotSame`.
+
+```import nomarkup
+PersonnummerNotSameExample.vue
+```
+
+Skriv så här i kod för att jämföra med ett inmatningsfält med `v-model="reference"`:
+
+```diff
+ <f-text-field
++    v-validation.personnummerFormat.personnummerNotSame="{ personnummerNotSame: { otherField: reference } }"
+ ></f-text-field>
+```
+
```

Comment:
Finns det ett generellt felmeddelande för validatorn? Skriv gärna in det här i texten. Om det inte finns utan konsumenten ska se till att det är på plats är det bra om den informationen också står här.
Samma sak för validatorerna nedan.

Reply:
Adderade raderna: 694, 714, 734.

Se nedan.